packages/apm/src/hubextensions.ts

```diff
@@ -26,43 +27,50 @@ function traceHeaders(): { [key: string]: string } {
  *
  * @param spanContext Properties with which the span should be created
  */
-function startSpan(spanContext?: SpanContext): Span {
+function startSpan(spanOrTransactionContext: SpanContext | TransactionContext): Transaction | Span {
   // @ts-ignore
   const hub = this as Hub;
   const scope = hub.getScope();
   const client = hub.getClient();
-  let span;
 
-  // This flag determines if we already added the span as a child to the span that currently lives on the scope
-  // If we do not have this, we will add it later on twice to the span recorder and therefore have too many spans
-  let addedAsChild = false;
+  let newSpanContext = spanOrTransactionContext;
 
+  // We create an empty Span in case there is no scope on the hub
+  let parentSpan = new Span();
```

Comment:
And why do we do that? W dont do anything with `parentSpan` later on.

Reply:
Good point, relic